presidio-anonymizer/README.MD

```diff
@@ -1 +1,126 @@
-# TODO add the document Omri wrote.
\ No newline at end of file
+# Presidio anonymizer
+
+## Description
+
+The Presidio anonymizer is a Python based module for anonymizing detected PII text
+entities with desired values.
+
+Presidio anonymizer comes by default with the following anonymizers:
+
+- Replace - replaces PII with desired value
+- Redact - removes PII completely from text
+- Hash - hash the PII using sha256
+- FPE - using ff1 algorithm for formatting-Preserving Encryption on the PII
+- Mask - mask the selected amount of characters of the PII with a given character
+
+[comment]: <shiranr> (#TODO need to fill this once the the functionality is done 2656 - 2659)
+Each anonymizer requires different values:
+
+- Replace -
+- Redact - None
+- Hash - None
+- FPE -
+- Mask -
+
+Text might have zero to many PIIs, the anonymizer handle those scenarios in the following
```

Comment:
```suggestion
As the input text could have potentially have overlapping PII entities, there are different anonymization scenarios:
```

Reply:
nit: Trying to rephrase to focus this on overlapping PII. Feel free to rephrase.

Reply:
DONE